doc/sphinx/changes.rst

```diff
@@ -8,6 +8,613 @@ Recent changes
+  (`#14813 <https://github.com/coq/coq/pull/14813>`_, by Laurent Théry).
+- **Added:**
+  The function ``sigT_of_prod`` turns a pair ``A * B`` into ``{_ : A & B}``. Its inverse function is ``prod_of_sigT``. This is shown by theorems ``sigT_prod_sigT`` and ``prod_sigT_prod``.
+  (`#14813 <https://github.com/coq/coq/pull/14813>`_, by Laurent Théry).
+- **Fixed:**
+  ``split_combine`` lemma for lists, making it usable
+  (`#14458 <https://github.com/coq/coq/pull/14458>`_,
+  by Yishuai Li).
+
+Infrastructure and dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Coq's continuous integration now provides a more accessible Windows
+  installer artifact in the "Checks" GitHub tab, both for pull
+  requests and the `master` branch.
+
+  This facilitates testing Coq's bleeding edge builds on Windows, and
+  should be more reliable than the previous setup (`#12425
+  <https://github.com/coq/coq/pull/12425>`_, by Emilio Jesus Gallego
+  Arias).
+- **Changed:**
+  Coq's ``./configure`` script has gone through a major cleanup. In
+  particular, the following options have been removed:
+  - ``-force-caml-version``, ``-force-findlib-version``: Coq won't
+  compile with OCaml or findlib lower than the required versions;
+  - ``-vmbyteflags``, ``-custom``, ``-no-custom``: linking options for
+  toplevels are now controlled in ``topbin/dune``;
+  - ``-ocamlfind``: Coq will now use the toolchain specified in the
+  Dune configuration; this can be controlled using the workspaces
+  feature;
+  - ``-nodebug``: Coq will now follow the standard, which is to always
+  pass ``-g`` to OCaml; this can be modified using a custom Dune workspace;
+  - ``-flambda-opts``: compilation options are now set in Coq's root
+  ``dune`` file, can be updated using a custom Dune workspace;
+  - ``-local``, ``-bindir``, ``-coqdocdir``, ``-annotate``,
+  ``-camldir``, ``-profiling``: these flags were deprecated in 8.14,
+  and are now removed.
+  Moreover, the ``-annot`` and ``-bin-annot`` flags only take effect
+  to set ``coq-makefile``'s defaults.
+  (`#14189 <https://github.com/coq/coq/pull/14189>`_,
+  by Emilio Jesus Gallego Arias).
+- **Changed:**
+  Configure will now detect the Dune version, and will correctly pass
+  ``-etcdir`` and ``-docdir`` to the install procedure if Dune >= 2.9 is available.
+  Note that the ``-docdir`` configure option now refers to root path for documentation.
+  If you would like to install Coq documentation in ``foo/coq``, use
+  ``-docdir foo``.
+  (`#14844 <https://github.com/coq/coq/pull/14844>`_,
+  by Emilio Jesus Gallego Arias).
+- **Changed:**
+  OCaml 4.13 is now officially supported
+  (`#14879 <https://github.com/coq/coq/pull/14879>`_,
+  by Emilio Jesus Gallego Arias)
+- **Changed:**
+  Sphinx 3.0.2 or above is now required to build the reference manual
+  (`#14963 <https://github.com/coq/coq/pull/14263>`_,
+  by Théo Zimmermann)
+
+Miscellaneous
```

Comment:
Since the two miscellaneous entries are about extraction, this could be renamed into "Extraction".